### ISSUE TITLE
Does the RDP Wrapper "fix" for Windows 10 Home still work?

### ISSUE THREAD

**Issue by puulima** [stascorp/rdpwrap#1068]
Seems like it hasn't been updated in a long time.  Just tried installing and RDP functionality is still not accessible.  Under the config program it notes the Wrapper State installed, ver 1.5.0.0, Service State is running - ver. 10.0.18362.657 but Listener State is  Not listening / not supported.

Been reading some posts and see a lot of references to updating the RDP Wrapper ini file?

Can somebody advise if this program / mod is still supported/still works.  And if so, what needs to be done to get it to work?  Much thanks

**Comment by robbak** [comment]
The program hasn't needed any updates, but the .ini files on this github and several others are regularly updated.

I recommend using the autoupdate.bat file in #859 . In addition, run the helper .bat file in the helper folder in that .zip file to schedule this script to run on each startup. As long as you don't run updates when they first appear, you'll always end up with a working .ini file before a new terminal services version hits.

**Comment by puulima** [comment]
I found the updated ini file and a video on how to manually update the ini and restart the RDPWrapper. Did all that and now when I run RDP Wrapper Config it shows the Service is STOPPED. Listener state still says NOT LISTENING but then off to the right it says "fully supported".
I've restarted using RDPWinst - r a few times, also rebooted... no luck

What am I missing?  Will running the "autoupdate" do anything different?
Thanks for the assistance...
